src/content/scripts/localization.js

```diff
@@ -0,0 +1,72 @@
+/*
+ license: The MIT License, Copyright (c) 2016-2019 YUKI "Piro" Hiroshi
+ original: http://github.com/piroor/webextensions-lib-l10n
+ 
+  Modification by John Bieling:
+   * Auto select ConversionHelper.i18n or browser.i18n
+   * Removed logging
+*/
+
+var localization = {
+  get i18n() {
+    if (!this._i18n) {
+      // Check wether this is called from the WebExtension part of the add-on,
+      // or from the legacy part, where the ConversionHelper provides the API.
+      // If the ConversionHelper is no longer needed, because the add-on has
+      // been converted to a pure WebExtension, this may simply return
+      // browser.1i8n
```

Comment:
line 17: browser => chrome

Reply:
Why chrome? The Firefox default is browser.*
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API
and will be present in any MDN documentation we link to.

We could go for messenger.* instead, but I did not want to introduce that here.

Reply:
messenger would be even better. I am just used to chrome because that is the namespace I am most familiar with. it's a short name for "the privileged context", traditionally (and has nothing to do with the same named 'chromium' browser from google - I have a feeling they "stole" that name from Mozilla) .

Reply:
Lets put this on the agenda of the next meeting. If I remember correctly, we decided to not use messenger.* but stick with browser.*

Using chrome.* would be very missleading, as WX is NOT privileged and has no access to "chrome" context (only experiments, which is a speciality). And it is the WX namespace from google chrome and friends...